src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -1782,6 +1844,104 @@ static void refda_adm_ietf_dtnma_agent_ctrl_ensure_odm(refda_ctrl_exec_ctx_t *ct
      * |START CUSTOM FUNCTION refda_adm_ietf_dtnma_agent_ctrl_ensure_odm BODY
      * +-------------------------------------------------------------------------+
      */
+
+    /*
+ari:/EXECSET/n=123;(//ietf/dtnma-agent/CTRL/inspect(//ietf/dtnma-agent/EDD/odm-list))
+821482187B8564696574666B64746E6D612D6167656E742267696E7370656374818464696574666B64746E6D612D6167656E7423686F646D2D6C697374
+
+ari:/EXECSET/n=123;(//ietf/dtnma-agent/CTRL/ensure-odm("test-org", 100, "test-model-1", -1))
```

Comment:
If the enumeration really doesn't matter, prefer to use the "example" org 65535, but not a big deal for isolated unit testing.